R/checks.R

```diff
@@ -161,19 +187,26 @@ check_file_details = function(relative_paths){
   return(not_found)
 }
 
-#' Check that the GAMBLR config you have loaded will work in your setup
+
+#' @title Check GAMBLR Config.
+#' 
+#' @description Check that the GAMBLR config you have loaded will work in your setup.
 #' 
-#' This function is mostly for remote GAMBLRs to ensure they keep their local mirror of the GAMBL data up-to-date. It 
+#' @details This function is mostly for remote GAMBLRs to ensure they keep their local mirror of the GAMBL data up-to-date.
 #'
 #' @param compare_timestamps Whether the function will compare timestamps on your local files to the remote copy. Only relevant if you are working remotely.
 #' @param ssh_session The ssh_session object see get_ssh_session() for more information. Only relevant if you are working remotely.
 #' @param archive_mode This is not currently working but the idea here is to keep a GSC archive of GAMBL in sync with the actively updated outputs
-#' @param force_backup 
+#' @param force_backup Boolean parameter set to FALSE per default.
 #'
 #' @return
+#' 
+#' @import dplyr tidyr
```

Comment:
would adding glue to this mean we don't have to put glue::glue everwhere?

Reply:
Yes, based on my testing. But I have not applied these changes globally. 

There are basically two ways to tell a function what packages are needed. First, tell the function under the import section (in the documentation). Or, directly specify the package when a function from an external package is being called, followed by double colons (i.e `package::function`). Both options are valid and should achieve the same thing, at least from the perspective of the main function. Not sure how it would affect the overall documentation. I am thinking that the second option would mean that the external package would not show up under imports in the NAMESPACE.